src/main/java/address/controller/RootLayoutController.java

```diff
@@ -119,6 +121,8 @@ private void handleSaveAs() {
         }
 
         modelManager.setPrefsSaveLocation(file.getPath());
+        // TODO temp patch to create cloud file if non found
+        raise(new CreateAddressBookOnRemoteRequestEvent(new CompletableFuture<>(), file.getName()));
```

Comment:
No, it does not, just like how GitHub doesn't automatically create a repo if we try to push to a non-existing one.